pegasus/src/homepage.rb

```diff
@@ -105,6 +105,20 @@ def self.get_heroes
   end
 
   def self.get_actions
+    # Show a Latin specific video to users browsing in Spanish (Mexico) to
+    # promote LATM HOC.
+    if I18n.locale == "es-MX"
+      youtube_id = ""
+      download_path = ""
+      facebook = ""
+      twitter = ""
```

Comment:
Still waiting on this info - the video isn't finished yet. I'll update when it's in.